### ISSUE TITLE
how can i install it in conda environment

### ISSUE THREAD

**Issue by CConory** [ytoon/Synchronized-BatchNorm-PyTorch#1]
how can i install it in conda environment?

**Comment by CConory** [comment]
i have solved the problem 
because stepup will use sudo command , it will change the env to the root_env and the python path also change
i make the python command  in root_path target to the anaconda python path, and it's worked.
